sdk/storage/src/clients/table_client.rs

```diff
@@ -0,0 +1,60 @@
+use crate::clients::TableServiceClient;
+use crate::table::requests::*;
+use azure_core::errors::AzureError;
+use http::method::Method;
+use http::request::{Builder, Request};
+use std::sync::Arc;
+
+pub trait AsTableClient<TN: Into<String>> {
```

Comment:
Why?

Reply:
Same as above, to implement a function for `Arc<Box<..>>`.